src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -99,6 +99,126 @@ public void run() {
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Ice, 2L),
                 200,
                 480);
+
+        // Fluid Sodium
+        GT_Values.RA.addFluidHeaterRecipe(
+                Materials.Sodium.getDust(1), GT_ModHandler.getSteam(1000), Materials.Sodium.getFluid(1000), 200, 120);
+
+        GT_Values.RA.addFluidSolidifierRecipe(
+                ItemList.Shape_Mold_Ball.get(0L),
+                Materials.Sodium.getFluid(1000),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 1L),
+                16,
+                4);
+        // Sodium Potassium
+        GT_Values.RA.addDistilleryRecipe(
```

Comment:
I'd like the old recipe back. There is a 2007 chinese patent making NaK by simply mixing molten sodium and potassium in an inert atomsphere, so it's feasible irl as well. This 1931 patent method is used irl at greater scale because pure potassium price is high. In GTNH pure potassium is cheap, so I'd prefer to use the 2007 patent method instead.

Reply:
but you think the mixer have this kind of Athmosphere ?  But the ratio need to be changed